nbextensions/usability/freeze/main.js

```diff
@@ -0,0 +1,126 @@
+define([
+	'base/js/namespace',
+	'base/js/events',
+	'notebook/js/codecell'
+], function (
+	Jupyter,
+	events,
+	codecell
+){
+	'use strict';
+	
+	var CodeCell = codecell.CodeCell;
+	
+	function patch_CodeCell_execute () {
+		console.log('[Freeze] patching CodeCell.prototype.execute')	
+		CodeCell.prototype.old_execute = CodeCell.prototype.execute;
+		
+		CodeCell.prototype.execute = function () {
+			if (!this.metadata.run_control.frozen) {
+				this.old_execute(arguments);
```

Comment:
This doesn't do what you (presumably) intended. `this.old_execute(arguments);` will pass the arguments as a list. You want `this.old_execute.apply(arguments)` - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/apply